src/theories/Category.jl

```diff
@@ -39,6 +39,34 @@ We use symbol ⋅ (\\cdot) for diagrammatic composition: f⋅g = compose(f,g).
   id(A) ⋅ f == f ⊣ (A::Ob, B::Ob, f::(A → B))
 end
 
+@theory coPresheaf{Ob,Hom,Psh} <: Category{Ob,Hom} begin
```

Comment:
Catlab follows the naming convention that theories are capitalized, so this should be `CoPresheaf` or `Copresheaf`. My preference is for the latter, following the [nLab](https://ncatlab.org/nlab/show/copresheaf).

Reply:
okay, sounds good.